### ISSUE TITLE
PHP 8 support

### ISSUE THREAD

**Issue by jgarifuna** [Sintattica/atk#72]
Hi, I think I saw PHP 8 support in one of the commits. Is PHP 8 officially supported yet?

**Comment by dangamba** [comment]
Hi Jorge,
we're still working on it.
If you'd like to try it, please let us know.

Daniele

Il giorno dom 2 giu 2024 alle ore 07:05 Jorge Garifuna <
***@***.***> ha scritto:

**Comment by bahtiarp** [comment]
I have patched the old ATK classic atk6 so can run in php8.3 with some AdminLte theme
also free qwab from @jgarifuna works as well.
i plan to publish in my github, but the old Atk Wiki , seems not working anymore.
Can i migrate the old wiki doc to my site?
Thanks.

**Comment by sanotto** [comment]
I have an old project in atk6 that could benefit from yur port...interested
in you pub...

**Comment by jgarifuna** [comment]
Hi @dangamba , I am interested in trying it out.

**Comment by dangamba** [comment]
> I have patched the old ATK classic atk6 so can run in php8.3 with some AdminLte theme also free qwab from @jgarifuna works as well. i plan to publish in my github, but the old Atk Wiki , seems not working anymore. Can i migrate the old wiki doc to my site? Thanks.

Hi @bahtiarp, if you want to host the old wiki and forum, we can send you php files and a dump of MySQL DBs.
Let me know if you're interested.

**Comment by dangamba** [comment]
> Hi @dangamba , I am interested in trying it out.

Hello Jorge, what exactly are you interested in trying out?

**Comment by bahtiarp** [comment]
> > I have patched the old ATK classic atk6 so can run in php8.3 with some AdminLte theme also free qwab from @jgarifuna works as well. i plan to publish in my github, but the old Atk Wiki , seems not working anymore. Can i migrate the old wiki doc to my site? Thanks.
> 
> Hi @bahtiarp, if you want to host the old wiki and forum, we can send you php files and a dump of MySQL DBs. Let me know if you're interested.

yes please. i'll host the old wiki and forum in my hosting server.
please tell me where should i download.
 🙏🙏🙏

**Comment by bahtiarp** [comment]
This is atkdemo on latest php 8.3.x and mysql 8.x
https://atkdemo.nusasms.com/
user: administrator/demo
I will publish the src when i got the old wiki/forum, so we can continue to maintain it.

**Comment by dangamba** [comment]
You can download forum and wiki sources at https://we.tl/t-b6aZOxLa5B
You'll find 2 files:
- www_root.tar.gz contains a index.php (that just redirects to this Github repo) and two folders, one for the wiki and one for the forum. It was meant to host them at /wiki and /forum routes respectively.
- mysql_dumps.zip contains a .sql file for each of the two databases.

If you please, I'll point the DNS of the atk-framework.com domain to your server.

Nice demo BTW

**Comment by jaskie79** [comment]
hi, i also am interested in trying out on an old atkframework project which i converted to laravel but its not the same...  there are news about it?

**Comment by dangamba** [comment]
the commit tagged v11 works with php 8.3

**Comment by sanotto** [comment]
trying it asap, thx you!!

**Comment by jgarifuna** [comment]
This is great news!

kind regards,

Jorge Garifuna
314-666-4402

**Comment by wayneh** [comment]
> This is atkdemo on latest php 8.3.x and mysql 8.x https://atkdemo.nusasms.com/ user: administrator/demo I will publish the src when i got the old wiki/forum, so we can continue to maintain it.

Any news on the new hosting of the Forum & Wiki ?
Thanks
Wayne

**Comment by bahtiarp** [comment]
@wayneh 
https://atk.nusasms.com/wiki/
@dangamba can you please point the original wiki url to: 103.78.210.226

thanks

**Comment by dangamba** [comment]
@bahtiarp done!

**Comment by bahtiarp** [comment]
https://atk-framework.com/wiki/ATK_Howtos
done!

**Comment by wayneh** [comment]
Thank you Bahtiar – well done !

 

I have a couple of applications using ATK 6  – containing some mods of mine and would like to move off PHP 5.6 :)

 

Your post said that you were using a patched version of ATK6 as per you demo page at https://atkdemo.nusasms.com/

 

Are you planning to release this ATK version ?

 

Thank you

 

Regards

Wayne

 

Wayne Hulls

20 Cornwall St

Vauxhall

DUNEDIN 9013

NEW ZEALAND

Mobile: +64 22 121 0015

**Comment by bahtiarp** [comment]
@wayneh 
yes, i'm planning to release it.

**Comment by wayneh** [comment]
GREAT !

 

Thank you Bahtia

 

Wayne

**Comment by bahtiarp** [comment]
@wayneh 
https://github.com/bahtiarp/NusaAtk

**Comment by sanotto** [comment]
This versión supports atk 6non php8 ?

**Comment by bahtiarp** [comment]
@sanotto 
I tried on PHP 7.4.33 it works.
Not try yet on php 5.6.

**Comment by sanotto** [comment]
Ok, I'll check it out.

**Comment by wayneh** [comment]
Thanks Bahtiar

 

I will convert one of my applications and let you know how I get on !

 

Thanks again

 

Wayne

**Comment by jaskie79** [comment]
can move on new repo? i tried on that but i have deprecation errors anyway.. do you? 
(in particular call to ereg() deprecated func)
ill post over new right now
anyway thanks to @bahtiarp and you all

**Comment by bahtiarp** [comment]
hi @dangamba i have upgrade old wiki to latest version of mediawiki.
can you pointing back atk-framework.com to: 103.78.210.226
temporarily wiki can be accesed: https://atkwiki.nusasms.com/
atkdemo php8 on: https://atkdemo.nusasms.com
thanks

**Comment by dangamba** [comment]
> hi [@dangamba](https://github.com/dangamba) i have upgrade old wiki to latest version of mediawiki. can you pointing back atk-framework.com to: 103.78.210.226 temporarily wiki can be accesed: https://atkwiki.nusasms.com/ atkdemo php8 on: https://atkdemo.nusasms.com thanks

Hello [@bahtiarp], done.
Please note that we'll keep the atkframework.com domain through 2026, but then we'll let it expire. If anyone is interested in transferring it, please DM me daniele[dot]gamba[at]gmail[dot]com

**Comment by bahtiarp** [comment]
Thanks @dangamba 

 I'm willing to continue managing the domain atk-frameork.com.
Please transfer the domain to me, i have send and email for the request.
